fixtures/case_entity.yaml

```diff
@@ -399,7 +399,8 @@ App\Entity\ResidentComplaintBoardCase:
   test-case-aarhus-1:
     leaseSize: 51
     complainant: Jens Jensen
-    complainantCPR: '1234567890'
+    complainantIdentifierType: CPR
+    complainantIdentifier: '123456-7890'
```

Comment:
Why are you adding `-`? The dash is not part of a CPR, but a part of displaying it.

Reply:
Fixed in 00acbb7fa8f6e10cb0b843f6d3ef8483c8a14dc3.